### DISCUSSION TITLE
How to have program check newegg for bundles

### DISCUSSION THREAD

**Discussion by Vuand** [jef/streetmerchant#1148] [Help]
### Description
Newegg seems to be starting to only sell items though bundles. Is there anyway you can have the program monitor the site for bundles as well?

**Comment by hectic-droid** [comment]
Go to the src/store/model folder and edit newegg.ts with your notepad (windows) or TextEdit (Mac).  Copy and paste any ps5 line of code that is present starting with the open bracket and ending with close bracket comma.  Then replace the url inside the quotation marks with the bundle url of your choosing.  Additionally, update the cart URL with the correct item number.  You can do this multiple times provided you use the open, close brackets and the comma.  The program will look for these items as well.  Remember that newegg may ban your IP address if you have the program refresh too often.  I have an example below on the coding as well as an updated newegg.ts with all three bundles in case you do not want to program yourself.
[newegg.ts.zip](https://github.com/jef/streetmerchant/files/5645511/newegg.ts.zip)



Example: 

               {
			brand: 'sony',
			cartUrl: 'https://secure.newegg.com/Shopping/AddtoCart.aspx?Submit=ADD&ItemList=N82E16868110292',
			model: 'ps5 console',
			series: 'sonyps5c',
			url:
				'https://www.newegg.com/p/N82E16868110292?Description=ps5&cm_re=ps5-_-68-110-292-_-Product'
		},

**Comment by Vuand** [comment]
Thanks so much droid, do you know if you have to include the **item number** or if you need only the **"cart url"** and **"url"** to have the bot check for the bundles? Also if you need the item number, where would you find that?

Thanks so much by the way

**Comment by jef** [comment]
> Thanks so much droid, do you know if you have to include the **item number** or if you need only the **"cart url"** and **"url"** to have the bot check for the bundles? Also if you need the item number, where would you find that?
> 
> Thanks so much by the way

`url` is required, it would have no other way to check. `cartUrl` and `itemNumber` are not.

Good luck!

https://github.com/jef/streetmerchant/wiki/Help:-Configuration:-Adding-a-store

**Comment by Vuand** [comment]
Alright, so there were a lot of drops with the newegg bundles around 4:00 pst today and the bot did not detect it. I'm seeing that the page is set up differently with the bundles compared to the pages selling only the graphics card. Any idea how to fix this?

**Comment by jef** [comment]
Do you mind sending me a couple bundles and I can give examples? Perhaps I can also add them.

Thanks!
